components/status/StatusActionsMore.vue

```diff
@@ -31,19 +31,23 @@ const { t } = useI18n()
 const userSettings = useUserSettings()
 const useStarFavoriteIcon = usePreferences('useStarFavoriteIcon')
 
+const isSingleInstanceServer = useRuntimeConfig().public.singleInstance
 const isAuthor = $computed(() => status.account.id === currentUser.value?.account.id)
+const notLocal = $computed(() => getServerName(status.account) !== currentServer.value)
```

Comment:
Better helper based on existing code ... still learning nuxt/elk so I appreciate the heads up @aaga